kraken-core/src/kraken/core/cli/main.py

```diff
@@ -321,13 +321,16 @@ def _print_task(task: Task) -> None:
             remaining_width -= 2
             if remaining_width <= 0:
                 remaining_width = width
+            first = True
             for part in textwrap.wrap(
```

Comment:
Nit: not a huge fan of boolean-flags within loop state. Perhaps we can be a bit more obvious with something like:

```python
parts = textwrap.wrap(description, remaining_width)
line.append(part[0]) # we only want to indent lines after the first
line.append("\n")
for part in parts[1:]:
    line.append((width - remaining_width - 1) * " ")
    line.append(part[0])
    line.append("\n")   
```

Just a style-choice overall, so not a blocker.

Reply:
Good call! Changed it to match this :)

Reply:
Another good way to solve this can be `for idx, part in enumerate(parts):` and checking for `idx == 0`. Though better suited if the code you would otherwise need to replicate in @tetigi's version is a but more. Here it definitely improves clarity to move the zero-index case out of the loop!